HPXMLtoOpenStudio/resources/EPvalidator.xml

```diff
@@ -726,12 +726,13 @@
   <sch:pattern name='[MechanicalVentilationType=CFIS]'>
     <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Systems/h:MechanicalVentilation/h:VentilationFans/h:VentilationFan[h:UsedForWholeBuildingVentilation="true" and h:FanType="central fan integrated supply"]'>
       <sch:assert test='count(h:AttachedToHVACDistributionSystem) = 1'>Expected 1 element(s) for xpath: AttachedToHVACDistributionSystem</sch:assert>
+      <sch:assert test='count(h:IsSharedSystem) = 0'>Expected 0 element(s) for xpath: IsSharedSystem</sch:assert>
     </sch:rule>
   </sch:pattern>
 
   <sch:pattern name='[MechanicalVentilationType=Shared]'>
     <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Systems/h:MechanicalVentilation/h:VentilationFans/h:VentilationFan[h:UsedForWholeBuildingVentilation="true" and h:IsSharedSystem="true"]'>
-      <sch:assert test='count(h:FractionRecirculation) = 1'>Expected 1 element(s) for xpath: FractionRecirculation</sch:assert>
+      <sch:assert test='count(h:FractionRecirculation) = 1 and number(h:FractionRecirculation) &gt;=0 and number(h:FractionRecirculation) &lt;=1'>Expected 1 element(s) value from 0.0 to 1.0 for xpath: FractionRecirculation</sch:assert>
```

Comment:
I haven't tested this, but I think it could simply be:
```suggestion
      <sch:assert test='count(h:FractionRecirculation[number(text()) &gt;= 0 and number(text()) &lt;= 1]) = 1'>Expected 1 element(s) for xpath: FractionRecirculation[number(text()) >= 0 and number(text()) <= 1]</sch:assert>
```